tfhe/src/core_crypto/algorithms/lwe_keyswitch.rs

```diff
@@ -146,6 +150,20 @@ pub fn keyswitch_lwe_ciphertext<Scalar, KSKCont, InputCont, OutputCont>(
     // Copy the input body to the output ciphertext
     *output_lwe_ciphertext.get_mut_body().data = *input_lwe_ciphertext.get_body().data;
```

Comment:
Could be merged with the new following condition

Reply:
what do you mean ?

Reply:
no because we need to copy the body no matter what the modulus is, if I move it we won't copy the body if moduli are equal

Reply:
It could be done in an else condition

Reply:
I don't think it's worth it here, we copy the body and round if needed for a single coefficient